src/main/java/spoon/support/reflect/code/CtConstructorCallImpl.java

```diff
@@ -165,20 +164,20 @@ public void removeArgument(CtExpression<?> argument) {
 
 	@Override
 	@DerivedProperty
-	public <T extends CtActualTypeContainer> T setActualTypeArguments(List<? extends CtTypeReference<?>> actualTypeArguments) {
+	public CtConstructorCall setActualTypeArguments(List<? extends CtTypeReference<?>> actualTypeArguments) {
```

Comment:
I ssuggest to use CtConstructorCallImpl, here and in all Ct*Impl implementations